structs.go

```diff
@@ -477,23 +508,7 @@ type GuildBan struct {
 
 // A GuildIntegration stores data for a guild integration.
 type GuildIntegration struct {
-	ID                string                   `json:"id"`
-	Name              string                   `json:"name"`
-	Type              string                   `json:"type"`
-	Enabled           bool                     `json:"enabled"`
-	Syncing           bool                     `json:"syncing"`
-	RoleID            string                   `json:"role_id"`
-	ExpireBehavior    int                      `json:"expire_behavior"`
-	ExpireGracePeriod int                      `json:"expire_grace_period"`
-	User              *User                    `json:"user"`
-	Account           *GuildIntegrationAccount `json:"account"`
```

Comment:
Guh...... We break the API here too.. I think maybe we just break the API, sorry to make you undo your change here, maybe just change Guild to use Integration and be done with it. Searching seems to show that noone is using GuildIntegration on github.

Reply:
Should I revert the last commit